src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20240909130645_EES5292_RemoveObsoleteGeospatialObjects.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    /// <inheritdoc />
+    public partial class EES5292_RemoveObsoleteGeospatialObjects : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable("geometry");
+            migrationBuilder.DropTable("geometry_columns");
+            migrationBuilder.DropTable("spatial_ref_sys");
+            migrationBuilder.Sql("DROP FUNCTION geometry2json");
```

Comment:
We should also delete file `GovUk.Education.ExploreEducationStatistics.Data.Model\Migrations\Current_Routine_geometry2json.sql` now.

We already deleted `Current_View_geojson` in https://github.com/dfe-analytical-services/explore-education-statistics/pull/4995/files#diff-4ae12b399586f9cf917a9530cad796bcd2c2a0023502126252fbb245be045e6e.